keyboards/atlantis/ak81_ve/ak81_ve.c

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2021
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#include "ak81_ve.h"
+
+#if !defined(VIA_ENABLE) && defined(ENCODER_ENABLE)
```

Comment:
```suggestion
#if defined(ENCODER_ENABLE)
```

Reply:
Not nescessary I know, but I think it makes the intention a little more clearer.. so I've decided to keep it in.